app/src/main/java/ch/epfl/sdp/map/HeatMapHandler.java

```diff
@@ -136,12 +137,15 @@ private void createGeoJson(@NotNull Map<String, Map<String, Object>> stringMapMa
 
         for (Map.Entry<String, Map<String, Object>> entry : entrySet) {
             try {
-                GeoPoint geoPoint = (GeoPoint) entry.getValue().get(GEOPOINT_TAG);
-                infectionHeatMapPoints.add(Point.fromLngLat(
-                        geoPoint.getLongitude(),
-                        geoPoint.getLatitude()
-                ));
+                if (entry.getValue().get(INFECTION_STATUS_TAG).equals("INFECTED")){
+                    GeoPoint geoPoint = (GeoPoint) entry.getValue().get(GEOPOINT_TAG);
+                    infectionHeatMapPoints.add(Point.fromLngLat(
+                            geoPoint.getLongitude(),
+                            geoPoint.getLatitude()
+                    ));
+                }
             } catch (NullPointerException ignored) {
+                List<Point> ddd = new ArrayList<>();
```

Comment:
What is this used for?

Reply:
It's not, I needed a statement for the debugger to stop on, Thanks !